### DISCUSSION TITLE
Development work currently thwarted due to being unable to use the donated money

### DISCUSSION THREAD

**Discussion by native-api** [pyenv/pyenv#2712] [General]
We're currently unable to withdraw any of the donated money from our fiscal host due to international sanctions -- since both our current active maintainers are in Russia. While our activity, of course, does not fall under the sanctions, many organizations, most notably major financial ones, have stopped dealing with any Russian entities indiscriminately.

This thwarts the development work since we can only afford to do so much for free. In particular, we had to put on hold all previously planned major improvements.
We however still reply to error reports and accept contributions.
Any bounties for implementing highly-requested features to persons outside Russia can likewise still be paid out.

We're working to resolve this issue but cannot give any ETA as we've already exhausted several options fruitlessly.

**Comment by leuasseurfarrelds247-arch** [comment]
[]()
